post-processor/googlecompute-export/post-processor.go

```diff
@@ -30,6 +30,7 @@ type Config struct {
 	Subnetwork          string   `mapstructure:"subnetwork"`
 	VaultGCPOauthEngine string   `mapstructure:"vault_gcp_oauth_engine"`
 	Zone                string   `mapstructure:"zone"`
+	ServiceAccountEmail string   `mapstructure:"service_account_email"`
```

Comment:
Ah; I'm sorry: I didn't see that one before; this will trigger the code generation to add some changes to our generated files ( mainly for our soon to come hcl2  conf). Runnin `make generate` will generate all the changes you need.